File: editor-ace/src/main/java/com/raredev/vcspace/editor/ace/VCSpaceWebChromeClient.java
package com.raredev.vcspace.editor.ace;

import android.util.Log;
import android.webkit.ConsoleMessage;
import android.webkit.WebChromeClient;

public class VCSpaceWebChromeClient extends WebChromeClient {

  @Override
  public boolean onConsoleMessage(ConsoleMessage consoleMessage) {
    Log.d(
        "VCSpace",
        consoleMessage.message()
            + " -- From line "
            + consoleMessage.lineNumber()
            + " of "
            + consoleMessage.sourceId());
    return true;
  }
}

File: editor-ace/src/main/java/com/raredev/vcspace/editor/ace/VCSpaceWebChromeClient.kt
package com.raredev.vcspace.editor.ace

import android.util.Log
import android.webkit.ConsoleMessage
import android.webkit.WebChromeClient

class VCSpaceWebChromeClient : WebChromeClient() {
  override fun onConsoleMessage(consoleMessage: ConsoleMessage): Boolean {
    Log.d(
        "VCSpace",
        "${consoleMessage.message()} -- From line ${consoleMessage.lineNumber()} of ${consoleMessage.sourceId()}")
    return true
  }
}
